dna-frontend/src/app/core/components/sidebar/sidebar.component.scss

```diff
@@ -0,0 +1,12 @@
+a {
```

Comment:
Same thing here
- Modify to classes instead of element tag

Reply:
made those changes